src/main/java/org/folio/circulation/domain/policy/Period.java

```diff
@@ -175,4 +190,11 @@ public int toMinutes() {
     }
   }
 
+  public boolean isLessThanOrEqualTo(Period otherPeriod) {
+    return toMinutes() <= otherPeriod.toMinutes();
+  }
+
+  public boolean isMoreThanOrEqualTo(Period otherPeriod) {
```

Comment:
Personally, it is more familiar to use `greater than` and `less than`, this might be a British English thing though. I don't mind if you don't change this.

```suggestion
  public boolean isGreaterThanOrEqualTo(Period otherPeriod) {
```